app/controller/InfoController.js

```diff
@@ -336,12 +336,18 @@ SDL.InfoController = Em.Object.create(
       web_app_frame.name = frame_name;
       web_app_frame.id = frame_name;
       web_app_frame.className = "WebEngineFrame";
+      web_app_frame.hidden = false;

```

Comment:
@YarikMamykin suggest to make hidden = true by default

Reply:
@YarikMamykin are you sure "hidden" makes exactly what you want?
Suggest to consider trick with "active_state" and "inactive_state" predefined styles in HMI project, used by most of views

Reply:
Here we should have hidden=false, because this method `runAppWithProperties` launches webApplication.